carma-messenger-ui/docker/install.sh

```diff
@@ -53,13 +53,12 @@ chmod +x /usr/local/bin/docker-compose
 # Configure user permissions for docker
 usermod -aG docker www-data
 
-# Old installtion for CARMA Platform remote start
 # Switch to dockerized version of launch and kill script
-# mv /var/www/html/scripts/docker_launch.bash /var/www/html/scripts/launch.bash
-# mv /var/www/html/scripts/docker_kill.bash /var/www/html/scripts/kill.bash
+ mv /var/www/html/scripts/docker_launch.bash /var/www/html/scripts/launch.bash
+ mv /var/www/html/scripts/docker_kill.bash /var/www/html/scripts/kill.bash
```

Comment:
If launch.bash and kill.bash are no longer used why not just call them directly in the php script? Extra step of renaming them seems a bit confusing.

Reply:
I have kill.bash and launch.bash version for my local VM. These two bash files are different from the docker versions. So my code calling kill.bash and launch.bash in local VM is different from the bash scripts in the docker container.

Reply:
It is also confusing if I call docker_launch bash in the code because it is not working if I do not use docker in my local VM

Reply:
It is addressed

Reply:
https://github.com/usdot-fhwa-stol/carma-messenger/commit/9ce44e4db69c5cd739fb7d11f8066accea044f76

Reply:
https://github.com/usdot-fhwa-stol/carma-messenger/commit/310935a176d7d970b80e57340d355b981ff7c84b